index.html

```diff
@@ -100,7 +95,108 @@ <h6>Skills</h6>
         <p class="left">MY RECENT WORK</p>
     </div>
     <hr>
-    
-    
+    <section class="works">
```

Comment:
- [ ] I think it is a good idea to have only one `<section>` tag for all the `works` section since we use the `<section>` tag for big content, but I can see that you have used it more than once in the works section, kindly let's fix this issue to keep the code more semantic 😉 
- [ ] Please remember that the text `MY RECENT WORK` is supposed to be inside the works section as well, since it's the title for the works section, for now, it has its own container outside of the works section, and also since it's a title you shouldn't use the `<p>` tag, you should use instead `hx` tags like `h2`, let's fix this please 👍

Reply:
Thank you.